vendor/k8s.io/apiserver/pkg/admission/plugin/webhook/namespace/matcher.go

```diff
@@ -93,7 +93,7 @@ func (m *Matcher) MatchNamespaceSelector(h *v1beta1.Webhook, attr admission.Attr
 		// namespace, it is exempted from all webhooks for now.
 		// TODO: figure out a way selective exempt cluster scoped resources.
 		// Also update the comment in types.go
-		return false, nil
```

Comment:
😢

Reply:
Yeah, we're going to have to update the code.  I'm thinking we apply the same label selector.

Reply:
Compare https://github.com/kubernetes/kubernetes/issues/57964, started discussion with @caesarxuchao there.